src/modules/dashboard/components/ColonyEvents/ColonyEvents.tsx

```diff
@@ -55,29 +69,62 @@ const ColonyEvents = ({ colony: { colonyAddress }, colony }: Props) => {
     (first: any, second: any) => {
       if (!(first && second)) return 0;
 
-      return eventsFilter === EventFilterOptions.NEWEST
+      return eventsSort === EventsSortOptions.NEWEST
         ? second.createdAt - first.createdAt
         : first.createdAt - second.createdAt;
     },
-    [eventsFilter],
+    [eventsSort],
+  );
+
+  const handleDataPagination = useCallback(() => {
+    setDataPage(dataPage + 1);
+  }, [dataPage]);
+
+  /* Needs to be tested when all event types are wirde up & reflected in the list */
+  const filtereEvents = useMemo(
+    () =>
+      !ethDomainId
+        ? events
+        : events.filter((event) => {
+            const displayValues = JSON.parse(event.displayValues);
+
+            return (
+              // Number(event.fundingPot) === ethDomainId ||
+              Number(event.domainId) === ethDomainId ||
+              Number(displayValues.fromPot) === ethDomainId ||
+              /* when transfering funds the list shows both sender & recipient domains */
+              (event.eventName ===
+                ColonyAndExtensionsEvents.ColonyFundsMovedBetweenFundingPots &&
+                Number(displayValues.toPot) === ethDomainId) ||
+              /* when no specific domain in the event it is displayed in Root */
+              (ethDomainId === 1 &&
+                event.domainId === null &&
+                event.fundingPot === undefined &&
+                displayValues.toPot === undefined &&
+                displayValues.fromPot === undefined)
+            );
+          }),
+    [ethDomainId, events],
```

Comment:
This is not needed here.

Consider the events list as a raw feed from the contract events.

That means no parsing of data what-so-ever, except for those that already come in the event `args`.

Please remove it